koans/about_scoring_project.rb

```diff
@@ -31,6 +31,22 @@
 
 def score(dice)
   # You need to write this method
+  val = 0
+  num_1 = dice.find_all{ |i| i == 1 }.count
+  num_2 = dice.find_all{ |i| i == 2 }.count
+  num_3 = dice.find_all{ |i| i == 3 }.count
+  num_4 = dice.find_all{ |i| i == 4 }.count
+  num_5 = dice.find_all{ |i| i == 5 }.count
+  num_6 = dice.find_all{ |i| i == 6 }.count
```

Comment:
what if will be [_d100_ dice](https://images-na.ssl-images-amazon.com/images/I/7157kwFUkvL._AC_SX425_.jpg)?
Use loops